src/status_im/contact/db.cljs

```diff
@@ -81,7 +81,7 @@
     (get contacts public-key
          (public-key->new-contact public-key))))
 
-(defn- contact-by-address [[_ contact] address]
+(defn- contact-by-address [contact address]
```

Comment:
wouldn't the subscription `:contacts/contact-by-address` also be affected by this change?

Reply:
it does, but the only usage of that sub is being made unused by this PR and will be removed anyway

Reply:
so could you remove it rather than keep it since it won't work anymore?

Reply:
If I start removing code, it'd start domino effect and this PR size will grow to from 21 to 50-something in size :(
We'll have one cleanup PR after all three parts have been merged (wallet, chat and dapps)